conf/broker.conf

```diff
@@ -957,8 +957,8 @@ managedLedgerCacheCopyEntries=false
 # Threshold to which bring down the cache level when eviction is triggered
 managedLedgerCacheEvictionWatermark=0.9
 
-# Configure the cache eviction frequency for the managed ledger cache (evictions/sec)
-managedLedgerCacheEvictionFrequency=100.0
+# Configure the cache eviction interval in milliseconds for the managed ledger cache
+managedLedgerCacheEvictionIntervalMs=0
```

Comment:
Should we use this new `managedLedgerCacheEvictionIntervalMs` instead of `managedLedgerCacheEvictionFrequency` in the default broker.conf?